low_level_platform/api/platform/lf_unix_clock_support.h

```diff
@@ -1,6 +1,8 @@
 #include <time.h>
 #include <errno.h>
 
+extern instant_t MIN_SLEEP_DURATION;
```

Comment:
Suggest renaming `lf_min_sleep_duration`.  Our convention uses upper case for `#define` constants` and the prefix `lf_` for all variables whose scope extends beyond one file.